EXTENSIONS.md

```diff
@@ -0,0 +1,236 @@
+# Possible extensions
+
+This file contains possible follow-on proposals which could be formulated within the framework of the class decorators proposal. This decorators proposal was designed to permit other syntactic elements to be decorated, but the Stage 2 proposal refers only to the class, field, method and accessor decorators. The main open question is whether to include parameter decorators initially, as they have attracted widespread interest and uptake in TypeScript within classes.
+
+## Annotation syntax
+
+**Annotations**, a declarative complement to decorators, use the syntax `@{ }`, which behaves exactly like an object literal. Arbitrary expressions, spread, computed property names, etc, are permitted. It is accessible from the annotated object through the `[Symbol.metadata]` property.
+
+```mjs
+@{x: "y"} @{v: "w"} class C {
+  @{a: "b"} method() { }
+  @{c: "d"} field;
```

Comment:
Does it possible to store non-primitive data?

```mjs
@{type: MyClass} field;
```

Reply:
Yes, anything value should work. Though I wonder if we want a more specialized mechanism for runtime types...